### PR TITLE
Add DTV map Musa Point

### PR DESCRIPTION
**Pull request by rf-l**
New DTV map Musa Point

### CODE REVIEW

File: src/Module.Client/GUI/MapSpriteData.xml [hunk 1]
```diff
@@ -142,6 +142,7 @@
       <SpriteSheetSize ID="134" Width="1920" Height="1080" />
       <SpriteSheetSize ID="135" Width="1920" Height="1080" />
       <SpriteSheetSize ID="136" Width="1920" Height="1080" />
+      <SpriteSheetSize ID="138" Width="1920" Height="1080" />
       <AlwaysLoad />
```
Comment (namidaka) [inline src/Module.Client/GUI/MapSpriteData.xml:146]: where's 137?

**Review by namidaka — CHANGES_REQUESTED** [review]

File: src/Module.Client/GUI/MapSpriteData.xml (hunk 1 above)
Comment (namidaka) [reply src/Module.Client/GUI/MapSpriteData.xml:146]: Volskullkeep has been merged , conflicts needs to be handled

**Review by namidaka — APPROVED** [review]

**Comment by rf-l** [comment]
Another commit for new map (Temple of Poseidon). Thought it would open a new PR. Since I thought it was merged.

**Comment by namidaka** [comment]
> Another commit for new map (Temple of Poseidon). Thought it would open a new PR. Since I thought it was merged.

image
This has never been properly rebased

**Comment by rf-l** [comment]
Ah, hope this fixed it.

**Pull request merged**